src/app/core/user/demo-user-generator.service.ts

```diff
@@ -11,6 +10,7 @@ import { LocalSession } from "../session/session-service/local-session";
 export class DemoUserGeneratorService extends DemoDataGenerator<User> {
```

Comment:
Not so easy because returning users and config objects at the same time means this needs to be filtered whenever somebody wants to access the demo user (e.g. in other demo data generators)